pkg/agent/datamodel/helper.go

```diff
@@ -48,6 +48,32 @@ func IsMIGNode(gpuInstanceProfile string) bool {
 	return gpuInstanceProfile != ""
 }
 
+// IsNvidiaEnabledSKU determines if an VM SKU has nvidia driver support.
+func IsNvidiaEnabledSKU(vmSize string) bool {
+	// Trim the optional _Promo suffix.
+	vmSize = strings.ToLower(vmSize)
+	vmSize = strings.TrimSuffix(vmSize, "_promo")
+	return NvidiaEnabledSKUs[vmSize]
+}
```

Comment:
Datamodel is very opinionated. Not sure how the new bootstrapping contract work is progressing but it might make sense to pull this out into a separate module. People who may not be interested in agentbaker's data model might be interested in gpu skus.

Reply:
part of me feels like this should be part of that skewer api

though maybe `pkg/agent/common` would be a good place to start?

Reply:
im also fine with leaving it in the datamodel for now if that's easiest, but Bryce has a point - it also begs the question of what exactly we want to be exposing from the agentbaker module in general

Reply:
Created package `pkg/agent/common`

Reply:
yes, also thinking if it's better for this to be part of the `skewer` API